src/__experimental__/components/input/input-presentation.spec.js

```diff
@@ -160,7 +165,7 @@ describe('InputPresentation', () => {
       assertStyleMatch({
         outline: 'none',
         border: '1px solid #255BC7'
-      }, renderWithTheme({ hasFocus: true }, classicTheme), {
+      }, renderWithTheme({ hasFocus: true }, classicTheme).find(InputPresentationStyle), {
```

Comment:
Does this theme here have to be `classic`? I'm just thinking if we can use a different theme here it means a bit less work to do when we come to removing the `classic` theme. No worries if not though.

Reply:
If changing this is trivial then go ahead but if it would involve significant changes to style assertions or chromatic snapshots I'd rather go with this as-is and change it as part of the classic deprecation please.

Reply:
@DipperTheDan To be honest I'd leave that until the `classicTheme` is removed from the `input-presentation` component

Reply:
That's fine and totally understand. Just figured I'd ask the question. Shy bairns get nothing 😉